ssh/tailssh/tailssh.go

```diff
@@ -51,6 +51,10 @@ var (
 	sshDisableSFTP       = envknob.RegisterBool("TS_SSH_DISABLE_SFTP")
 	sshDisableForwarding = envknob.RegisterBool("TS_SSH_DISABLE_FORWARDING")
 	sshDisablePTY        = envknob.RegisterBool("TS_SSH_DISABLE_PTY")
+
+	// terminalError is an empty PartialSuccessError, which results in the ssh
```

Comment:
... is an empty PartialSuccessError (with no 'Next' authentication methods that may proceed), which results ...

and "SSH server" (since it's in prose)

... "immediately disconnecting the client without sending the 'Partial' bit on the wire"